actionpack/lib/abstract_controller/caching/fragments.rb

```diff
@@ -135,6 +135,7 @@ def expire_fragment(key, options = nil)
       end
 
       def instrument_fragment_cache(name, key) # :nodoc:
+        return yield unless ActionView::Base.enable_fragment_cache_logging
```

Comment:
When I said I config on Action View I didn't remember the fragment caches were in Action Pack. So we can't do this, it couples Action Pack to Action View.

Look to where `perform_caching` is defined in Action Pack, and move the config itself there.